src/NServiceBus.Persistence.DynamoDB.Tests/NServiceBus.Persistence.DynamoDB.Tests.csproj

```diff
@@ -11,8 +11,6 @@
     <PackageReference Include="GitHubActionsTestLogger" Version="2.0.1" />
     <PackageReference Include="AWSSDK.DynamoDBv2" Version="3.7.101.56" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.5.0" />
```

Comment:
```suggestion
    <!-- DO NOT REMOVE Newtonsoft.Json: Added due to an implicit dependency from Particular.Approvals. This can be removed once Particular.Approvals has been updated to 13.0.3 -->
    <PackageReference Include="Newtonsoft.json" Version="13.0.3" />
```